test/ethclient.go

```diff
@@ -1664,6 +1666,11 @@ func (c *Client) AuctionEventInit(genesisBlockNum int64) (*eth.AuctionEventIniti
 	}, 1, nil
 }
 
+// GetCoordinatorsLibP2PAddrs required for the interface
+func (c *Client) GetCoordinatorsLibP2PAddrs() ([]multiaddr.Multiaddr, error) {
+	return nil, errors.New("TODO")
```

Comment:
what this todo error means?

Reply:
This is the implementation of an interface that is used for testing. Since we're not maintaining tests done this way, I left this as a TODO, because I needed to add the method otherwise the code wouldn't compile